web-app/src/main/webapp/css/style.css

```diff
@@ -25,6 +25,7 @@
   background: #fffcfc;
   border: 2px solid #9e9e9e;
   border-radius: 6px;
+  color: #6e6e6e;
```

Comment:
protip: devtools allows you to check if your font colors have enough contrast for accessibility